.github/workflows/shisen_ci.yml

```diff
@@ -0,0 +1,23 @@
+name: Shisen CI
+on:
+  pull_request:
+    branches:
+      - master
+  push:
+    branches:
+      - master
+      
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+    name : Build the Project
+
+    steps:
+    - uses: actions/checkout@v2
+    - uses: actions/setup-node@v2
+      with:
+        node-version: '14'
+    - run: npm install
+    - run: npm test
```

Comment:
Run the `npm run build` as specified in the [`package.json`](https://github.com/ichiro-its/shisen-rvue/blob/master/package.json). Run it after the `npm install` and before the `npm test`.